nixos/tests/all-tests.nix

```diff
@@ -901,3 +902,4 @@ in {
   zrepl = handleTest ./zrepl.nix {};
   zsh-history = handleTest ./zsh-history.nix {};
 }
+
```

Comment:
there should only be one trailing newline

Reply:
Thanks, solved